js/src/constants.ts

```diff
@@ -0,0 +1,43 @@
+// https://www.jsonrpc.org/specification#compatibility
+export const JSON_RPC_VERSION = '2.0';
+export type JSON_RPC_VERSION_TYPE = typeof JSON_RPC_VERSION;
+
+// http://www.jsonrpc.org/specification#error_object
+// http://xmlrpc-epi.sourceforge.net/specs/rfc.fault_codes.php
+export const PARSE_ERROR = -32700;
+export const INVALID_REQUEST = -32600;
+export const METHOD_NOT_FOUND = -32601;
+export const INVALID_PARAMS = -32602;
+export const INTERNAL_ERROR = -32603;
+
+// Rockets error codes
+export const INVALID_JSON_RESPONSE = -31001;
+export const REQUEST_ABORTED = -31002;
+export const HTTP_ERROR = -31003;
+
+// Rockets server notifications
+export const CANCEL = 'cancel';
+export const PROGRESS = 'progress';
+
+export const ROCKETS_WS_PROTOCOL = 'rockets';
```

Comment:
this is actually imposed by an application afaik, we chose `rockets` for brayns, but Rockets server itself has no predefined subprotocol. Without knowing all implications, we should probably just expose this one to the user to specify that.

Reply:
Done